MLH-Fellowship/VehiCO
# Landing Page

## Description
Current Behavior:

- Hello World landing page

Proposed Changes:

- Implemented landing page according to design in #4 
- Added autocomplete for origin and destination inputs
- Added debounce time to limit query calls (when the user is done typing)
- Send lat & lon data to /map route

### Issue Number(s):
Closes #10 

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/37096589/101150820-a192f100-35d5-11eb-92a4-738856a79614.png)


## How Has This Been Tested?



** Make sure to add Geoapify API key to .env file **

Manual testing steps:
1. Input an origin address
2. Input a destination address
3. Press the confirm button
4. Confirm that the lat/lon is printed in the console for origin and destination
5. An alert dialog should pop up if an origin or destination is not provided

## Pull request type
 
Please check the type of change your PR introduces:

## Checklist
Please check if your PR fulfills the following requirements:
- [x] Docs have been reviewed and added / updated if needed
- [x] My code follows the code style of this project and lint has passed locally
